packages/wrangler/src/core/register-commands.ts

```diff
@@ -3,6 +3,7 @@ import chalk from "chalk";
 import { fetchResult } from "../cfetch";
 import { readConfig } from "../config";
 import { FatalError, UserError } from "../errors";
+import { demandSingleValue } from "../index";
```

Comment:
can you move this function to another util so this module doesn't pull in all of wrangler

Reply:
and demandOneOfOption while you're at it – I thought this had already been moved 🤔

Reply:
Moved to `./core/helpers.ts` https://github.com/cloudflare/workers-sdk/pull/7005/commits/19ebe94780c4bbffa3bd450229238d32a0bad7fb